src/java/org/apache/cassandra/repair/messages/RepairOption.java

```diff
@@ -436,24 +442,34 @@ public boolean ignoreUnreplicatedKeyspaces()
         return ignoreUnreplicatedKeyspaces;
     }
 
+    public boolean repairData()
+    {
+        return repairData;
+    }
+
     public boolean repairPaxos()
     {
         return repairPaxos;
     }
 
     public boolean paxosOnly()
     {
-        return paxosOnly;
+        return !repairAccord && !repairData;
+    }
+
+    public boolean repairAccord()
+    {
+        return repairAccord;
     }
 
     public boolean accordOnly()
     {
-        return accordOnly;
+        return !repairPaxos && !repairData;
     }
 
     public boolean isConsensusOnly()
     {
-        return paxosOnly() || accordOnly();
+        return (repairPaxos() || repairAccord()) && !repairData;
```

Comment:
for consistency with the above logic, this should probably just be `!repairData` ?

Reply:
That's slightly less defensive? `!repairData` doesn't imply that accord or pacts are being repaired.